src/py/tests/test_page_generator.py

```diff
@@ -104,6 +104,7 @@
 """)
 
 
+@pytest.mark.order(1)
```

Comment:
this might seem out of place, but since some tests rely on `import plotly`, this has to run before that occurs, since it tests the idea that `plotly` is not available when kaleido runs